client/src/components/AdvancedSearch.jsx

```diff
@@ -0,0 +1,191 @@
+import React, { useState, useEffect, useRef } from "react";
+import { FormControl, FormGroup, FormLabel } from 'react-bootstrap';
+import "../Search.css";
+import ListGroup from "react-bootstrap/ListGroup";
+import ListGroupItem from "react-bootstrap/ListGroupItem";
+import { PencilSquare } from "react-bootstrap-icons";
+import { Link } from "react-router-dom";
+import _ from "lodash";
+
+/* Import Components */
+import Input from "../components/Input";
+import Button from "../components/Button";
+
+import '../containers/FormContainer.css'
+
+const { REACT_APP_PROXY } = process.env;
+
+let abortController = new window.AbortController();
+
+const initCoopSearchSettings = () => {
+	return {
+  name: '',
+  type: '',
+  street: '',
+  city: '',
+  state: '',
+  zip: '',
+  enabled: true,
+  };
+};
+
+const doSearch = (coopSearchSettings, setSearchResults, setLoading) => {
+	// logic is very similar to doSearch in Search components
+	// except searchUrl is built with multiple parameters from completed form
+  abortController.abort();
+  abortController = new window.AbortController();
+  setLoading(true);
+
+	// build search url
+	let searchUrl = REACT_APP_PROXY + '/coops/';
+	if (coopSearchSettings.name != '') {
+		searchUrl = searchUrl + "?name=" + encodeURIComponent(coopSearchSettings.name);
+	}
+	if (coopSearchSettings.enabled != 'none') {
+		searchUrl = searchUrl + "?enabled=" + encodeURIComponent(coopSearchSettings.enabled);
```

Comment:
In the event they search with both name and enabled flags, "?" probably won't work since you already have that on line 42.  Just need a little logic here to switch it to "&" if there is already a search param included.

Reply:
Thank you Dave, good catch!  I added logic to ensure that the initial "?" is only added before the first search parameter and that "&"s are added after all later parameters.